scripts/install/install-dependencies.sh

```diff
@@ -0,0 +1,394 @@
+    fi
+  else
+    # Install wide field indices. If we change to using longer focal length
+    if [[ ! -f index-4116.fits ]] ; then
+      curl --remote-name \
+           http://broiler.astrometry.net/~dstn/4100/index-411[6-9].fits
+    fi
+  fi
+}
+
+#-------------------------------------------------------------------------------
+
+function show_help() {
+  echo "${THIS_PROGRAM} - Install software needed for PANOPTES."
+  echo " "
+  echo "${THIS_PROGRAM} [options]"
+  echo " "
+  echo "options:"
+  echo "-h, --help                 show brief help"
+  echo "-x                         turn on bash debug output"
+  echo "--run <function>           run the named function and exit, for debugging"
+  echo "--no-apt-get               don't run apt-get to install Linux packages"
+  echo "--no-mongodb               don't install and start mongodb server"
+  echo "--no-cfitsio               don't install the latest version of cfitsio"
+  echo "--no-conda                 don't install the latest version of Anaconda"
+  echo "--no-astrometry            don't install astrometry.net software"
+  echo "--no-astrometry-indices    don't install astrometry.net indices"
+  echo "--astrometry-full-indices  install full indices, for narrow and wide fields"
+  echo "--no-pip-requirements      don't install python packages"
+}
+
+cd "${PANDIR}"
+mkdir -p tmp
+
+while test ${#} -gt 0; do
+  case "${1}" in
+    -h|--help)
+      show_help
+      exit 0
+      ;;
+    -x)
+      set -x
+      shift
+      ;;
+    --run)
+      if [ $# -lt 2 ]; then
+        echo "Function name missing after ${1}."
+        show_help
+        exit 1
+      fi
+      shift
+      (${1})
+      exit
+      ;;
+    --no-apt-get)
+      DO_APT_GET=0
+      shift
+      ;;
+    --no-mongodb)
+      DO_MONGODB=0
+      shift
+      ;;
+    --no-cfitsio)
+      DO_CFITSIO=0
+      shift
+      ;;
+    --no-conda)
+      DO_CONDA=0
+      shift
+      ;;
+    --no-astrometry)
+      DO_ASTROMETRY=0
+      shift
+      ;;
+    --no-astrometry-ind*)
+      DO_ASTROMETRY_INDICES=0
+      DO_ASTROMETRY_FULL_INDICES=0
+      shift
+      ;;
+    --astrometry-full-ind*)
+      DO_ASTROMETRY_FULL_INDICES=1
+      DO_ASTROMETRY_INDICES=1
+      shift
+      ;;
+    --no-pip-requirements)
+      DO_PIP_REQUIREMENTS=0
+      shift
+      ;;
+    *)
+      echo "Unknown parameter: ${1}"
+      echo
+      show_help
+      exit 1
+      ;;
+  esac
+done
+
+
+if [[ "${DO_APT_GET}" -eq 1 ]] ; then
+  install_apt_packages
+fi
+
+
+if [[ "${DO_MONGODB}" -eq 1 ]] ; then
+  maybe_install_mongodb
+fi
+
+
+if [[ "${DO_CONDA}" -eq 1 ]] ; then
+  install_conda_if_missing
+fi
+
+
+if [[ "${DO_CFITSIO}" -eq 1 ]] ; then
+  install_latest_cfitsio
+fi
+
+
+if [[ "${DO_ASTROMETRY}" -eq 1 ]] ; then
+  (install_astrometry)
+fi
+
+
+if [[ "${DO_ASTROMETRY_INDICES}" -eq 1 ]] ; then
+  (install_astrometry_indices)
+fi
+
+
+if [[ "${DO_PIP_REQUIREMENTS}" -eq 1 ]] ; then
```

Comment:
Let's actually create a separate panoptes environment in conda and use that. I've just started calling mine `panoptes-env`. So we will want to create it, install pip items there, and optionally set it as the default environment for the system (via `~/.profile`).  Something like:

```bash
$ conda create -y -n panoptes-env python=3
# Wait
$ source activate panoptes-env
(panoptes-env) $ pip install -U pip
(panoptes-env) $ pip install -r requirements.txt
(panoptes-env) $ echo "source activate panoptes-env" >> ~/.profile # Assume we always want
```
Note also we can create a conda environment file that can do all this and have the right collection of files. That is what I was doing in the consolidated branch but was having hassle. We should get this install script working first and then later worry about the environment file. But I do think we want to create/use a specific panoptes environment as outlined above.

Reply:
Why do you say "# Wait" after conda create?

Reply:
I just meant that it takes a minute and there is output between those two steps.

Reply:
Done (well, no wait included).